cmd/zpool/zpool_main.c

```diff
@@ -2119,6 +2119,17 @@ show_import(nvlist_t *config)
 				    "updating.\n"));
 				break;
 
+			case ZPOOL_ERRATA_ZOL_6845_ENCRYPTION:
+				(void) printf(gettext(" action: existing "
+				    "encrypted datasets cannot be used with "
+				    "this\n\tversion of zfs due to an on-disk "
+				    "incompatibility which\n\tneeds to be "
+				    "corrected. Revert to an eariler"
+				    "version and\n\tbackup and destroy any "
+				    "existing encrypted datasets\n\tbefore "
+				    "updating.\n"));
```

Comment:
this doesn't seem consistent with the PR description - can these old encrypted datasets be mounted readonly?  If so then it shouldn't be necessary to revert to an earlier version.

Reply:
This message needs to up updated, version zero encrypted datasets can be mounted read-only.  How about something like this which is inline with the [information posted about the errata](https://github.com/zfsonlinux/zfsonlinux.github.com/pull/35/commits/6c66084451420a3f2a792404b65564ce55bf01fd).

```
action: The pool can be imported but due to an on-disk incompatibility existing
        encrypted datasets can only be mounted read-only.  After import affected
        datasets can be be converted to the new version by using 'zfs send' and
        'zfs receive'.
```